realm/realm-library/src/main/cpp/io_realm_internal_SharedRealm.cpp

```diff
@@ -37,7 +37,7 @@ Java_io_realm_internal_SharedRealm_nativeInit(JNIEnv *env, jclass, jstring tempo
 JNIEXPORT jlong JNICALL
 Java_io_realm_internal_SharedRealm_nativeCreateConfig(JNIEnv *env, jclass, jstring realm_path, jbyteArray key,
         jbyte schema_mode, jboolean in_memory, jboolean cache, jboolean disable_format_upgrade,
-        jboolean auto_change_notification, jstring sync_server_url, jstring sync_user_token)
+        jboolean auto_change_notification, jstring sync_server_url, jstring /*sync_user_token*/)
```

Comment:
I can remove `sync_user_token` if it is not going to be used by the OS sync manager.